tests/src/unit-capacity.cpp

```diff
@@ -416,7 +416,7 @@ TEST_CASE("capacity")
     {
         SECTION("boolean")
         {
-            json j = true;
+            json const j = true;
             const json j_const = true;
```

Comment:
This is one of those cases I mentioned on Discord. This should use `NOLINT` instead.